backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/OggInputStream.java

```diff
@@ -454,10 +451,14 @@ public int read (byte[] b, int off, int len) {
 			if (value >= 0) {
 				b[i] = (byte)value;
 			} else {
-				if (i == 0) return -1;
-				return i;
+				if (i == 0) {
+					return -1;
+				} else {
+					return i;
+				}
```

Comment:
This didn't really need to change, though it's immaterial.

Reply:
This is from copying LWJGL3's changes into LWJGL2, as are the other immaterial changes in that class. I'm not too bothered either way - just feels like life would be easier if there's not too much divergence between them.

https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/audio/OggInputStream.java#L463